src/test/kotlin/io/openfuture/state/util/DummyData.kt

```diff
@@ -1,33 +1,29 @@
 package io.openfuture.state.util
 
+import io.openfuture.state.blockchain.Blockchain
 import io.openfuture.state.controller.WalletController
 import io.openfuture.state.domain.Transaction
 import io.openfuture.state.domain.Wallet
 import org.bson.types.ObjectId
+import org.mockito.Mockito
 import java.time.LocalDateTime
-import java.util.*
 
 fun createDummyWallet(
         id: ObjectId = ObjectId(),
         address: String = "address",
         webhook: String = "webhook",
-        transactions: Set<Transaction> = setOf(createDummyTransaction()),
+        blockchain: Blockchain = createDummyBlockchain(),
         lastUpdate: LocalDateTime = LocalDateTime.now()
-) = Wallet(address, webhook, transactions, lastUpdate, id)
+) = Wallet(address, webhook, blockchain.getName(), lastUpdate, id)
 
 fun createDummyTransaction(
         hash: String = "hash",
-        externalHash: String = "external hash",
-        typeId: Int = 1,
         participant: String = "participant address",
         amount: Long = 100,
         fee: Long = 0,
-        date: Long = Date().time,
+        date: LocalDateTime = LocalDateTime.now(),
         blockHeight: Long = 1,
         blockHash: String = "block hash"
-) = Transaction(hash, externalHash, typeId, participant, amount, fee, date, blockHeight, blockHash)
+) = Transaction(hash, participant, amount, fee, date, blockHeight, blockHash)
 
-fun createDummySaveWalletRequest(
-        address: String = "address",
-        webhook: String = "webhook"
-) = WalletController.SaveWalletRequest(address, webhook)
+fun createDummyBlockchain() = Mockito.mock(Blockchain::class.java)
```

Comment:
In the `DummyData.kt` we are having only wide-spread models. 

Mocks are tests specific and should not be here, but should be in a test definition. As well as test's specific models.